### PR TITLE
Circle 2 and Confit Changes

### PR DESCRIPTION
**Pull request by forstermatth**
@autovance/developers Here is a new config for using Circle 2 and the newest confit settings, if you're interested.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    docker:
+        - image: circleci/node:8
+    steps:
+      - checkout
+      - restore_cache:
+          key: npm-install-{{ checksum "package.json" }}
+      - run:
+          name: Test
+          command: npm run test:coverage
+          environment:
```
Comment (trs) [inline .circleci/config.yml]: Extraneous

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+version: 2
+jobs:
+
+  build:
+    docker:
+      - image: circleci/node:6
+    steps:
+      - checkout
```
Comment (trs) [inline .circleci/config.yml]: Can we cache `checkout`?

File: .circleci/config.yml (hunk 1 above)
Comment (kobayashik) [reply .circleci/config.yml]: superfluous

File: .circleci/config.yml (hunk 2 above)
Comment (forstermatth) [reply .circleci/config.yml]: It doesn't seem like it, and it seems like it should be an indiscernible change.

**Pull request merged**